serverlessworkflow/antora.yml

```diff
@@ -6,6 +6,7 @@ nav:
 - "modules/ROOT/nav.adoc"
 asciidoc:
   attributes:
+    kogito_version: 1.23.0.Final
```

Comment:
I think this will come from the released tag. This doc will be aligned with Kogito releases, no need to hardcode it. @akumar074 can you confirm, please?

Reply:
@akumar074 @ricardozanini are we going to keep the version here or?